<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_METER_ARCH_DATA_SETTTINGS</class>
 <widget class="QWidget" name="widget_METER_ARCH_DATA_SETTTINGS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Период сохранения (мин)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="singleStep">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Кратность выравнивания архива</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_agiment_time">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>нет выравнивания</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>минута</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>час</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>день</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Параметр 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_param_id_MDT_1"/>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox_param_id_tariff_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox_param_id_phase_1"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_param_id_phase_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_param_id_MDT_2"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Параметр 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_param_id_MDT_3"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_param_id_phase_3"/>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBox_param_id_tariff_3"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Параметр 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_param_id_tariff_2"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Параметр 4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_param_id_MDT_4"/>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="comboBox_param_id_phase_4"/>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="comboBox_param_id_tariff_4"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
